clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11823,6 +11823,9 @@ def err_sycl_external_no_rdc : Error<
 def warn_sycl_kernel_too_big_args : Warning<
   "size of kernel arguments (%0 bytes) may exceed the supported maximum "
   "of %1 bytes on some devices">, InGroup<SyclStrict>, ShowInSystemHeader;
+def warn_esimd_kernel_too_big_args : Warning<
+  "size of esimd kernel arguments (%0 bytes) may exceed the supported maximum "
```

Comment:
Is this diagnostic necessary?
I'm not sure it is worth adding new diagnostic just to add `esimd` word. In fact, I think the old message is perfectly fine for `esimd` kernels, isn't it?